docker/epoch_node1_mean15.yaml

```diff
@@ -21,10 +21,9 @@ mining:
     beneficiary_reward_delay: 2
     autostart: true
     cuckoo:
-        miner:
```

Comment:
this changes was not very good idea, because now docker-compose (master) cannot be used with stable (i.e. 1.1.0) docker image.